src/components/Button/index.module.scss

```diff
@@ -0,0 +1,136 @@
+@use '../../styles/variables' as *;
+
+%hover-style {
+  background-color: $color-light-blue-1;
+  color: $color-dark-blue-1;
+}
+
+.largeButtonStyle {
+  background-color: #658286;
+  border-color: #658286;
```

Comment:
you can add this to the global variable file or make it into a variable in this file if it not being used in other styles